packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/message-channel.object-metadata.ts

```diff
@@ -71,6 +71,14 @@ export class MessageChannelObjectMetadata extends BaseObjectMetadata {
   })
   type: string;
 
+  @FieldMetadata({
+    type: FieldMetadataType.BOOLEAN,
+    label: 'Is Contact Auto Creation Enabled',
+    description: 'Is Contact Auto Creation Enabled',
+    icon: 'IconUserCircle',
```

Comment:
You can set a defaultValue here FYI (so you don't have to put it in the insert in google-gmail.service.ts